nemo/export/trt_llm/nemo/nemo_ckpt_convert.py

```diff
@@ -1,647 +0,0 @@
-# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-
-import configparser
-import json
-import logging
-import math
-import multiprocessing
-import os
-import typing
-from collections import defaultdict
-from pathlib import Path
-from typing import Union
-
-import numpy as np
-import tensorstore  # This is important even though not used. Otherwise zarr raises error.
-import torch
-import zarr
-from tensorrt_llm._utils import np_bfloat16, pad_vocab_size, str_dtype_to_torch, torch_to_numpy
-from torch.distributed.checkpoint import FileSystemReader, TensorStorageMetadata
-from torch.distributed.checkpoint.state_dict_loader import load_state_dict
```

Comment:
@oyilmaz-nvidia I can see a few imports are gone in the new `nemo_file`, leading to NameErrors: `TensorStorageMetadata`, `load_state_dict`, `np_bfloat16`. Can you bring them back please?

(Also `GPT2Tokenizer` is missing but I don't know if this one is intended)